### PR TITLE
Kirti pereira 1199438 feedback file update

### PR DESCRIPTION
**Pull request by KirtiPereira**
Updated the Feedback file as per the WorkItem ID 1199438. Request review.

### CODE REVIEW

**Review by laujan — COMMENTED** [review]
Hi @KirtiPereira. Although Markdown supports inline HTML, HTML isn't recommended for publishing to Microsoft Docs. Can you please update the HTML tags to MarkDown formatting? See https://docs.microsoft.com/en-us/contribute/markdown-reference#html for general guidance and https://docs.microsoft.com/en-us/contribute/markdown-reference#bold-and-italic-text specifically for bold and italic text.

File: msteams-platform/feedback.md [hunk 1]
```diff
@@ -4,28 +4,24 @@ description: The Microsoft Teams product team responds to the developer communit
...
+| **Features suggestions**             |      Please make your features suggestions on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
+|        **Documentation issues**        |                                                                                                                                                                      Click `This page` option in the **Submit and view feedback for** section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repo.                                                                                                                                                                      |
```
Comment (laujan) [inline msteams-platform/feedback.md:19]: Don't use the term "click" See https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/click

File: msteams-platform/feedback.md [hunk 2]
```diff
@@ -4,28 +4,24 @@ description: The Microsoft Teams product team responds to the developer communit
...
+|       **Documentation updates**        | Click the Edit link on an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repo.                                                                                                                                                                      |
```
Comment (laujan) [inline msteams-platform/feedback.md:20]: Re: "Click" same as above.

File: msteams-platform/feedback.md (hunk 1 above)
Comment (KirtiPereira) [reply msteams-platform/feedback.md]: Updated the text as per the style-guide.

File: msteams-platform/feedback.md (hunk 2 above)
Comment (KirtiPereira) [reply msteams-platform/feedback.md]: Done.

**Pull request merged**